macOS/LocalPackages/NetworkProtectionMac/Sources/NetworkProtectionUI/SwiftUI/MenuItemButton.swift

```diff
@@ -20,23 +20,41 @@ import Foundation
 import SwiftUI
 
 struct MenuItemButton: View {
+
+    private struct Detail {
```

Comment:
Would it be beneficial to add a comment describing what information is included in the Detail?

Reply:
Will add.

Reply:
Sorry just adding a note here because I merged before adding that comment.  My apologies.

Will follow up as I agree with the suggestion.